### DISCUSSION TITLE
Repeat #2964

### DISCUSSION THREAD

**Discussion by jukrb0x** [mantinedev/mantine#2963] [Feature requests]
Repeat #2964

**Comment by foxxycodes** [comment]
Please don't try to get more attention on your posted topics. @rtivital is answering at an incredible pace (your topic #2964 got answered within 24h). Please have some respect for the maintainers and value their time properly.

**Reply by jukrb0x** [reply]
thanks for your reminder, this post is repeatedly sent due to network jam, so there were two exactly same posts. I've edited this post earlier to try not to get attention because I found there is no way to delete this post, if you would pay attention to the datetime you will find it out, and now this post is showing up at the top of the discussion with your comment :) ...

**Reply by jukrb0x** [reply]
welcome to delete this post if any maintainer is checking this out, thanks!
